package.json

```diff
@@ -5,15 +5,29 @@
   "dependencies": {
     "@material-ui/core": "^3.5.1",
     "@material-ui/icons": "^3.0.1",
+    "connected-react-router": "^6.0.0",
+    "cryptocurrency-icons": "^0.9.3",
+    "dotenv": "^6.2.0",
+    "history": "^4.7.2",
+    "qrcode.react": "^0.8.0",
+    "query-string": "^6.2.0",
     "react": "^16.6.3",
+    "react-compose": "^2.0.0",
     "react-dom": "^16.6.3",
-    "react-scripts": "2.1.1"
+    "react-redux": "^6.0.0",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
+    "react-scripts": "2.1.1",
+    "recompose": "^0.30.0",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0"
```

Comment:
are you planning on using thunk or saga ?

Reply:
we are planning on using sagas

Reply:
- [ ] Remove redux-thunk